source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
  "amount"
  "description"
  "reference"
- "agreement_id"
- }
- ```
+ "mandate_id"
+}
+```
 
 Once a mandate has been set up and a mandate ID is available, you can make an
 API call to collect a payment against the mandate.
 
-You do not need to wait until the mandate is confirmed to send a collection
-request to GOV.UK Pay.
+You do not need to wait until the mandate is confirmed to send a collection request to GOV.UK Pay.
 
-When collecting a payment against a mandate, you must [supply a
-Reference number](#reference-numbers) and description.
-
-If a mandate does not exist prior to a payment request, it takes 4 working
+If a mandate does not exist prior to a payment request, it takes 3 working
```

Comment:
Depending on how you look at it, the mandate does exist, it’s just not been set up with the bank yet.

Reply:
@alexbishop1 Just to check I've understood this - what status would the mandate be if it hadn't been set up with the bank (ie a 3-day turnaround)? And what status would it be if it *had* been set up (ie a 2-day turnaround).

Reply:
I checked with @katiebates and I've updated this in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c so it's hopefully clearer.